ssh/client.go

```diff
@@ -48,7 +48,6 @@ const (
 var (
 	baseSSHArgs = []string{
 		"-o", "PasswordAuthentication=no",
-		"-o", "IdentitiesOnly=yes",
```

Comment:
Does this have any effect on the other drivers / providers?  /cc @nathanleclaire @sthulb